### PR TITLE
Styling/icons

### PR DESCRIPTION
**Pull request by Braun-Johannes**

### CODE REVIEW

File: utils/index.js [hunk 1]
```diff
@@ -12,3 +14,18 @@ export function formatPetsAge(seconds) {
   parts.push(`${seconds}s`);
   return parts.join(" ");
 }
+
+export function SVGIcon({ variant, color = "black", size = 50, ariaLabel }) {
```
Comment (doemser) [inline utils/index.js]: this is not a utils function, it is a react function component and should be treated as it

File: components/CurrentPet.jsx [hunk 2]
```diff
@@ -1,10 +1,13 @@
 import StyledList from "@/components/Styles/StyledList";
 import StyledListItem from "@/components/Styles/StyledListItem";
+import { SVGIcon } from "@/utils";
 
 export default function CurrentPet({ selectedPet }) {
   return (
     <StyledList>
-      <StyledListItem $variant="select">{selectedPet.type}</StyledListItem>
+      <StyledListItem $variant="select">
+        <SVGIcon variant={selectedPet.type} size={300} />
```
Comment (doemser) [inline components/CurrentPet.jsx:9]: Probably better to call props names starting with "$" else you are passing useless attributes to the dom

**Review by doemser — CHANGES_REQUESTED** [review]
Good one, but place the icon component in the components folder

File: components/PetList.jsx [hunk 3]
```diff
@@ -21,7 +22,7 @@ export default function PetList({ onSelectPet, selectedPet }) {
                   onSelectPet(pet);
                 }}
               >
-                {pet.type}
+                <SVGIcon $variant={pet.type} size={100} ariaLabel={pet.type} />
```
Comment (doemser) [inline components/PetList.jsx]: I am sorry I left the comment for the prop name prefixes with "$" which only applies to styled components. As you are not even using styled components on the SVGIcon.. it is safe to say, that I was completely wrong :D

**Review by doemser — APPROVED** [review]

**Review by evabergold — APPROVED** [review]

**Pull request merged**